ophyd/controls/ophydobj.py

```diff
@@ -319,6 +319,17 @@ def check_value(self, value, **kwargs):
         '''
         pass
 
+    @property
+    def state(self):
+        return {}
+
+    def configure(self, state=None):
+        # does nothing; subclasses can override if configuration is possible
+        return self.state, self.state
+
+    def deconfigure(self):
+        return self.state
+
```

Comment:
I meant to link `Base`: https://github.com/NSLS-II/bluesky/blob/master/bluesky/examples.py#L10